binscripts/gvm-installer

```diff
@@ -103,7 +103,13 @@ if [ -z "$GVM_NO_UPDATE_PROFILE" ] ; then
   elif [ "$(uname)" == "Linux" ]; then
     update_profile "$HOME/.bashrc" || update_profile "$HOME/.bash_profile"
   elif [ "$(uname)" == "Darwin" ]; then
-    update_profile "$HOME/.profile" || update_profile "$HOME/.bash_profile"
+    LOGIN_SHELL=$(finger $(id -u -n) | grep Shell | cut -d : -f 3)
```

Comment:
```suggestion
    LOGIN_SHELL=$(finger -m $(id -u -n) | grep Shell | cut -d : -f 3)
```
I have multiple usernames that start with my name, like "bob", "bob-work", "bob-hidden". The -m command matches exactly, instead of potentially getting additional matches.